frontend/src/components/PolicyandTerms/index.jsx

```diff
@@ -0,0 +1,12 @@
+import styles from "./PolicyAndTerms.module.css";
```

Comment:
nazwę pliku `PolicyandTerms.module.css` trzeba też zmienić na pascalcase
`PolicyAndTerms.module.css`

jak i nazwę folderu PolicyandTerms na PolicyAndTerms

Reply:
zmieniłam, nie widać tych zmian?

Reply:
na moim branchu są zmienione
mam odpowiedź - nothing to commit, working tree clean